### PR TITLE
feat: stats in sidebar

### PR DESCRIPTION
**Pull request by chrisrummel**

### CODE REVIEW

**Comment by razonyang** [comment]
Thanks for share your ideas back to this theme. I'll start a review later.

File: layouts/partials/sidebar/stats.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{{- if (default true .Site.Params.showStats) -}}
+  <section class="stats row card component">
+    <div class="card-header">
+      <h2 class="card-title">{{ i18n "string_stats" }}</h2>
```
Comment (razonyang) [inline layouts/partials/sidebar/stats.html:4]: Please use `stats_*` instead, such as `stats_title`.

File: layouts/partials/sidebar/stats.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    </div>
+    <div class="card-body">
+      {{ i18n "string_posts" }}: {{ len .Site.RegularPages }}<br />
```
Comment (razonyang) [inline layouts/partials/sidebar/stats.html:7]: `stats_posts`.

File: layouts/partials/sidebar/stats.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+      {{ i18n "string_last_update" }}: {{ now.Format $.Site.Params.dateFormat }}
```
Comment (razonyang) [inline layouts/partials/sidebar/stats.html:8]: `stats_last_update`.

**Review by razonyang — CHANGES_REQUESTED** [review]

**Comment by chrisrummel** [comment]
feat: stats in sidebar (requested changes)

File: layouts/partials/sidebar/stats.html (hunk 3 above)
Comment (chrisrummel) [reply layouts/partials/sidebar/stats.html]: I tried to change everything, but I am not sure if I did it correctly.

**Comment by chrisrummel** [comment]
Sorry, I don't get it. The tests fail at `$.Site.Params.dateFormat` - which is nothing I have changed anything about. The initial PR deployed just fine, and this one, although nothing changed here (besides the i18n-strings), doesn't.

**Pull request closed without merging**